src/store/reducers/appReducer.js

```diff
@@ -67,7 +67,8 @@ const initialState = {
   deviceGroups: {},
   activeDeviceGroupId: undefined,
   theme: 'dark',
-  version: '0.0.1' // TODO: Version should be requested from the service
+  version: '0.0.1', // TODO: Version should be requested from the service
+  azureMapsKey: 'v_r1uH9qfLTJZijJdGbXMt_YHCVPQeXqsgFd2Xs_lpg'
```

Comment:
This shouldn't really be checked in, right? Because now, everyone can see it.
Or is there no reason to protect it like our other connection string type keys?

Reply:
No it should not, removed.